pipelines/optimus/adapter.wdl

```diff
@@ -42,6 +42,7 @@ task GetInputs {
     Array[String] r1_fastq = read_lines("r1.txt")
     Array[String] r2_fastq = read_lines("r2.txt")
     File i1_file = "i1.txt"
+    String timestamp = read_string("timestamp.txt") # this is a hack to force disabling the task level call-caching
```

Comment:
Instead of modifying pipeline-tools to create a timestamp file, can you instead create it in the command block?

Since the timestamp usage is a "hack" to work around the call caching behavior, and is not part of the pipeline itself (e.g. it is adaptor only), it seems like it should only be in the adapter code.

One way to do this in bash is to print out the seconds since the epoch, e.g.
```
date +%s > timestamp.txt
```

Reply:
This is a great idea and I will switch to it!